Source/Scene/Cesium3DTile.js

```diff
@@ -276,8 +259,40 @@ define([
     /**
      * DOC_TBA
      */
-    Cesium3DTile.prototype.isRefinable = function() {
-        return this._numberOfUnrefinableChildren === 0;
+    Cesium3DTile.prototype.isRefinable = function(cullingVolume) {
+        //>>includeStart('debug', pragmas.debug);
+        if (!defined(cullingVolume)) {
+            throw new DeveloperError('cullingVolume must be defined');
+        }
+        //>>includeEnd('debug');
+
+        var visibleChildren = [];
+        var child;
+        var k;
+
+        for (k = 0; k < this.children.length; ++k) {
```

Comment:
Usually we create the length variable outside of the loop, like:

```javascript
var length = this.children.length;
for (k = 0; k < length; ++k) {
```

This applies to other areas in the PR.